lib/mdl/rules.rb

```diff
@@ -214,7 +214,8 @@
 rule 'MD013', 'Line length' do
   tags :line_length
   aliases 'line-length'
-  params :line_length => 80, :code_blocks => true, :tables => true
+  params :line_length => 80, :ignore_code_blocks => false, :code_blocks => true
+  params :tables => true
```

Comment:
```suggestion
  params :line_length => 80, :ignore_code_blocks => false, :code_blocks => true,
         :tables => true
```

Reply:
You are right. I guess I did not push all my changes the last time I pushed to the branch.